apps/reward_histories/resources.py

```diff
@@ -101,16 +110,16 @@ def get(self):
     @userRequired
     def post(self):
         """Post a new data to reward history table
+
         Args:
```

Comment:
Karena semua argumen didapat dari JSON, lebih baik jika pada bagian Args-nya ditambahkan keterangan located in JSON, menjadi "Args (located in JSON)"